openownership/data-standard
# requirements: Update dependencies

# Overview

- What does this pull request do?

Update python dependencies

- How can a reviewer test or examine your changes?

Check that docs build okay, and there are formatting issues

- Who is best placed to review it?

Closes #

## Translations

      translating. See: https://openownership.github.io/bods-dev-handbook/translations.html

## Documentation & Release

- [x ] I've checked that the documentation builds without failing. See: https://github.com/openownership/data-standard#building-the-documentation-locally
      https://openownership.github.io/bods-dev-handbook/standard_releases.html
